CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ option(MUE_BUILD_NOTATION_MODULE "Build notation module" ON)
 option(MUE_BUILD_NOTATION_TESTS "Build notation tests" ON)
 option(MUE_BUILD_PALETTE_MODULE "Build palette module" ON)
 option(MUE_BUILD_PLAYBACK_MODULE "Build playback module" ON)
+option(MUE_BUILD_ENGRAVING_PLAYBACK "Build engraving playback" ON)
```

Comment:
Please move this option above, right after the other MUE_BUILD_ENGRAVING options